subjects/java/checkpoints/age-finder/README.md

```diff
@@ -0,0 +1,60 @@
+## Age Finder
+
+### Instructions
+
+Create a class `AgeFinder` that provides a method to calculate the age from a given date. The date will be provided in the format `yyyy-MM-dd`.
+
+In case of any error the method `calculateAge` should return `-1`
+
+> 💡 Be careful with the future.
```

Comment:
This doesn't say a lot :sweat_smile: It's not clear what should happen since the `int` return from the function can be negative (even if probably it won't make sense). I would add, more than "Be careful with the future", something like "Age can only be a positive number or zero" (I don't know about 0)